octoprint_octolapse/settings.py

```diff
@@ -2054,10 +2054,11 @@ def to_dict(self, ):
             'rendering_output_format_options': [
                 dict(value='avi', name='AVI'),
                 dict(value='flv', name='FLV'),
-                dict(value='vob', name='VOB'),
+                dict(value='gif', name='GIF'),
+                dict(value='h264', name='H.264'),
```

Comment:
This should also say MP4 - H.264, or something else that indicates the file type.